website/docs/reference/resource-configs/delimiter.md

```diff
@@ -0,0 +1,102 @@
+---
+resource_types: [seeds]
+datatype: <string>
+default_value: ","
+---
+
+## Definition
+
+An optional seed configuration used to customize how you separate values in a [seed](/docs/build/seeds) with the one-character string you provide.
+
+* The delimiter defaults to a comma when not specified.
+* Explicitly set The `delimiter` configuration value if you want seed files to use a different delimiter, such as "|" or ";".
```

Comment:
Same feedback here as above - is this feature changing the seed file or telling dbt how to interpret it?